jenkins-scripts/docker/ignition-abichecker.bash

```diff
@@ -47,6 +47,19 @@ then
   export NEED_C17_COMPILER=true
 fi
 
+# check if OGRE-2.2 include paths are needed
+if [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gazebo"   && ${IGN_NAME_PREFIX_MAJOR_VERSION} -ge 6 ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gui"       && ${IGN_NAME_PREFIX_MAJOR_VERSION} -ge 6 ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-sensors"   && ${IGN_NAME_PREFIX_MAJOR_VERSION} -ge 6 ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-rendering" && ${IGN_NAME_PREFIX_MAJOR_VERSION} -ge 6 ]]
```

Comment:
`ign-launch` may also be needed

Reply:
https://github.com/ignition-tooling/release-tools/pull/563